tutorials/MD/03_Running_an_MD_simulation.ipynb

```diff
@@ -0,0 +1,315 @@
+    "\n",
+    "`% cp /home/shared/data/1hsg.pdb .` \n",
+    "\n",
+    "If you look at this file (using the graphical editors vi or nedit for example, you should immediately see that it has two chains; A and B:\n",
+    "\n",
+    "`% vi 1hsg.pdb`\\\n",
+    "or\\\n",
+    "`% nano 1hsg.pdb`\n",
+    "\n",
+    "Now we need to prepare our protein for simulation. First of all we will extract only the protein coordinates from the pdb file into a new file called `protein.pdb`. To do this enter the following command:\n",
+    "\n",
+    "`% grep ATOM 1hsg.pdb > protein.pdb`\n",
+    "\n",
+    "Use your preferred text editor to open the protein.pdb file and see how it differs from the 1hsg.pdb file.\n",
+    "\n",
+    "We now need to generate the parameter/topology file we need. This process will also make sure that all the hydrogens are added to our protein. \n",
+    "\n",
+    "`% gmx pdb2gmx -f protein.pdb -ignh -o protein.gro`\n",
+    "\n",
+    "The `-ignh` flag will ignore any hydrogen atoms that might already be present in the .pdb file.\n",
+    "\n",
+    "The program should run and present a list of force-fields from which to select. Select the AMBER99SB-ILDN force field which should be option 6 in the list followed by 1 to select the recommended TIP3P water. If all goes well this should generate several files:\n",
+    "\n",
+    "1. topol.top \n",
+    "2. topol_Protein_chain_A.itp \n",
+    "3. topol_Protein_chain_B.itp \n",
+    "3. posre_Protein_chain_A.itp\n",
+    "4. posre_Protein_chain_B.itp\n",
+    "5. protein.gro\n",
+    "\n",
+    "Type:\n",
+    "\n",
+    "`% ls`\n",
+    "\n",
+    "to verify that all the above files have been created and are in your directory.\\\n",
+    "Note that the protein has a net charge of +4e. You should see a line that says \"Total charge\n",
+    "in system 4.000 e\".\n",
+    "\n",
+    "Before we can add water we need to define a box in which to put the protein and the water:\n",
+    "\n",
+    "`% gmx editconf -f protein.gro -box 7 7 7 -c -o boxed.gro`\n",
+    "\n",
+    "This puts the protein in the centre of the box that is 7 nm x 7 nm x 7 nm and creates the resulting file boxed.gro. As you may remember from the lecture, in this setup we will apply periodic boundary conditions, which means that the box is replicated indefinitely in all three directions. \n",
+    "\n",
+    "Next we need to add water to the system. We can add the water by repeatedly overlaying a small box of water into the system (216 molecules).\n",
```

Comment:
> by repeatedly overlaying a small box of water into the system (216 molecules).

At least to me this is a little bit confusing, it probably needs to be clarified that this is what `solvate` does?

Reply:
Rephrased